src/less/tables.less

```diff
@@ -46,7 +44,10 @@
         transition: ease-in-out 0.1s background-color;
       }
 
-      &:hover:not([disabled]):not(.disabled) {
+      /* stylelint-disable-next-line selector-max-specificity*/
+      &:hover:not([disabled]),
+      &:hover:not(.disabled) {
```

Comment:
I know I suggested this, but I'm not sure it'll work actually.

Reply:
Imma change it then.